### DISCUSSION TITLE
Blank Screen Android v9

### DISCUSSION THREAD

**Discussion by mijaelsaban** [quasarframework/quasar#14992] [CLI - Capacitor / Cordova modes]
Hi 

I have a blank screen after the splash screen on android v9 with the error, I have no problems on android v11 or v12:
```
Error:
11-27 16:17:34.294 I/chromium( 1508): [INFO:CONSOLE(0)] "Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "". Strict MIME type checking is enforced for module scripts per HTML spec.", source: file:///android_asset/www/assets/index.e64cc177.js (0)
```

Versions:
Cordova 9.0.0
"quasar": "^2.6.0",
"vue-router": "^4.0.0"
"vue": "^3.0.0",

Any help please?

Thank you.

**Comment by metalsadman** [comment]
Try upgrading your quasar deps `quasar upgrade -i`.
